inst/EpiModel2.rmd

```diff
@@ -376,15 +434,18 @@ need to be updated to the two-group syntax, noting that to replicate the biparti
 ## EpiModel 2.0 
 num1 <- num2 <- 500
 
-# update to new initialization function network_initialize.
+## update to new initialization function network_initialize.
 nw <- network_initialize(n = num1 + num2)
 
-# Update to new vertex assignment function set_vertex_attribute
+##Update to new vertex assignment function set_vertex_attribute
 nw <- set_vertex_attribute(nw, "group", rep(1:2, c(num1,num2)))
 
-# If looking to replicate 'bipartite' network:
+## If looking to replicate 'bipartite' network:
 formation <- ~edges + nodematch("group")
 target.stats <- c(400, 0)
+
+## If interested in within group mixing; here 100 of the total edges match on group:
```

Comment:
to clarify, does this mean 100 total edges match within group 1 and group 2 equally? like 50 in group 1 and 50 in group 2? 

```suggestion
## If interested in within-group mixing - here 100 of the total edges match in group:
```

Reply:
This means there are 100 edges that match on the "group" attribute, whether in group 1 or group 2.